src/index.html

```diff
@@ -1,45 +1,42 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Leaderboard</title>
     <link rel="stylesheet" href="./index.css">
 </head>
+
 <body>
     <header>
         <h1>Leaderboard</h1>
     </header>
 
     <main class="main-container">
-            <section class="part1">
-                <div class="header1">
-                    <h2>Recent scores</h2>
-                    <button class="refresh-btn"><b>Refresh</b></button>
-                </div>
-    
-                <div class="listOfNames">
-                    <ul>
-                        <li>Name <span>:100</span></li>
-                        <li>Name <span>:20</span></li>
-                        <li>Name <span>:50</span></li>
-                        <li>Name <span>:78</span></li>
-                        <li>Name <span>:125</span></li>
-                        <li>Name <span>:77</span></li>
-                        <li>Name <span>:42</span></li>
-                    </ul>
-                </div>
-            </section>
-    
-            <section class="part2">
-                <h2>Add your score</h2>
-                <form class="check-in">
-                    <input type="text" name="Name" maxlength="30" placeholder="Your name" required>
-                    <input type="number" name="Scores" placeholder="Your scores" required>
-                    <button type="submit">Submit</button>
-                </form>
+        <section class="part1">
+            <div class="header1">
+                <h2>Recent scores</h2>
+                <button class="refresh-btn" id="click" ><b>Refresh</b></button>
+            </div>
+
+            <div class="listOfNames">
+                <ul id="ListOfScores">
+                    
+                </ul>
+            </div>
+        </section>
+
+        <section class="part2">
+            <h2>Add your score</h2>
+            <form class="check-in" id="form">
+                <input type="text" name="user" maxlength="30" placeholder="Your name" required>
+                <input type="number" name="score" placeholder="Your scores" required>
+                <button type="submit">Submit</button>
+            </form>
```

Comment:
![image](https://github.com/Wils-FOURR-speed/Leaderboard-project/assets/93039359/8b56735e-9941-420d-96c6-a600a8640e79)

**[OPTIONAL]**

- [ ] It would be nice if you make your form clear after inserting the data into the API to give a great user experience 👍

Reply:
Okay sure. I'll do that later on.